src/depfile_parser_test.cc

```diff
@@ -132,13 +132,10 @@ TEST_F(DepfileParserTest, Escapes) {
   // Put backslashes before a variety of characters, see which ones make
   // it through.
   string err;
-  EXPECT_TRUE(Parse(
-"\\!\\@\\#$$\\%\\^\\&\\[\\]\\\\:",
-      &err));
+  EXPECT_TRUE(Parse("\\!\\@\\#$$\\%\\^\\&\\[\\]\\\\::", &err));
```

Comment:
Hm ... why change this input? Maybe add another test instead.

Reply:
I believed this test to be quite fitting as it's the only test I could find that deals with character contains. It also already tested for correct escaping of # which is treated the same way as : in the linked GCC source code.

Reply:
Sure, but : is a bit special and I like having the same input to see exactly how the parser changed. By changing this test I had to double check what exactly is going on.

Also: Doesn't the double-colon have a special meaning?

Please add a new test which explicitly checks with a real world scenario like `C\:/msys64/mingw32/i686-w64-mingw32/include/stdio.h`.

Reply:
You're right.

Added a test that uses two paths from depfiles of a project of mine that has once been created by GCC 10 and once by Clang